relbench/external/graph.py

```diff
@@ -87,7 +87,10 @@ def make_pkey_fkey_graph(
 
         if len(col_to_stype) == 0:  # Add constant feature in case df is empty:
             col_to_stype = {"__const__": stype.numerical}
-            df = pd.DataFrame({"__const__": np.ones(len(table.df))})
+            pkey_fkey_dict = {key: df[key] for key in table.fkey_col_to_pkey_table}
+            if table.pkey_col:
+                pkey_fkey_dict[table.pkey_col] = df[table.pkey_col]
```

Comment:
what does it do? Add pkey as feature?

Reply:
If we don't include fkey's, we will hit a bug at [this line](https://github.com/snap-stanford/relbench/blob/main/relbench/external/graph.py#L113)

Reply:
I see! good catch! Do you mind adding an informative comment on why we do this?

Reply:
Sure